src/patterns/validation/index.md.njk

```diff
@@ -39,6 +39,12 @@ Validation should refuse to accept:
 
 For example, if you’re asking for someone’s date of birth you should not accept ‘13’ in the month field.
 
+Remove or ignore any whitespace in users’ answers where it’s not needed. If an invisible character causes a validation error, it can be hard for users to know what's gone wrong and how to fix it. Consider removing:
+
++ spaces entered as part of numbers and codes, such as postcodes or card details
++ leading or trailing whitespace that a user may have been accidentally copied and pasted
++ extra spaces inserted by dictation software – this is particularly common when dictating numbers
+
```

Comment:
Suggested edit, taking in everyone's comment in PR and issue so far.

```suggestion
Validation should help remove (or ignore) any spaces or invisible characters, unless it makes the information too ambiguous for you to use.

If spaces or invisible characters cause a validation error, it's difficult for the user to see and fix them —especially if they got added by accident.

Consider removing spaces and invisible characters entered:
+ as part of numbers and codes, such as postcodes or card details
+ before or after an answer, as a user might have accidentally copied and pasted this in
+ by dictation software — this is particularly common when dictating numbers
```

Reply:
Thanks for that @calvin-lau-sig7 – reads nicely to me!